src/bubbleSort.js

```diff
@@ -0,0 +1,14 @@
+export default function bubbleSort(array) {
+  if(array.length < 2) return array
```

Comment:
I recommend the use of explicit blocks with control flow in order to increase readability.